package.json

```diff
@@ -7,12 +7,13 @@
     "start": "parcel",
     "build": "rm -rf dist; parcel build --detailed-report",
     "test": "playwright test",
+    "check": "tsc --noEmit",
     "fmt": "prettier --write .",
     "fix": "prettier --write scripts/ src/ tests/; eslint --fix scripts/ src/ tests/",
     "lint": "prettier --check . && eslint scripts/ src/ tests/",
-    "add-city": "node scripts/add-city.js",
-    "update-city-boundaries": "node scripts/update-city-boundaries.js",
-    "update-lots": "node scripts/update-lots.js",
+    "add-city": "ts-node-esm scripts/add-city.ts",
```

Comment:
Need to use ts-node-esm to run `ts` scripts

Reply:
Gave it a quick try and had trouble with the loader? (apparently I needed to use the experimental loader to run these files).  Didn't feel like digging into it so I'm sticking with ts-node-esm.